### ISSUE TITLE
我的cdua版本是10.0，tensorflow-gpu版本是1.13.1，但是cudnn没有找到与cunda10.0对应的cudnn7.4版本，有人遇到相同问题吗

### ISSUE THREAD

**Issue by Jess1989** [bubbliiiing/mask-rcnn-keras#13]

**Comment by Jess1989** [comment]
![image](https://user-images.githubusercontent.com/67063323/115978127-0dd7b680-a5b0-11eb-94bc-cb3e7f78b779.png)
只找到了7.6的，没有找到7.4的。有没有遇到这样问题的可以帮助一下。

**Comment by bubbliiiing** [comment]
cudnn不应该直接在英伟达官网下载吗0 0
